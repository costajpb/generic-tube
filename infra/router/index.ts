import { createRouter, createWebHistory } from 'vue-router'
import DashboardView from '@/infra/views/DashboardView.vue'

const router = createRouter({
  history: createWebHistory(import.meta.env.BASE_URL),
  routes: [
    {
      path: '/',
      name: 'dashboard',
      component: DashboardView
    },
    {
      path: '/shows/:id',
      name: 'show-details',
      component: () => import('@/infra/views/ShowDetailsView.vue')
    }
  ]
})

export default router
