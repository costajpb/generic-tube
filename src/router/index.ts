import { createRouter, createWebHistory } from 'vue-router'
import DashboardView from '@/src/views/DashboardView.vue'

const router = createRouter({
  history: createWebHistory(import.meta.env.BASE_URL),
  routes: [
    {
      path: '/',
      name: 'dashboard',
      component: DashboardView
    },
    {
      path: '/shows/:id',
      name: 'show-details',
      component: () => import('@/src/views/ShowDetailsView.vue')
    }
  ]
})

export default router
