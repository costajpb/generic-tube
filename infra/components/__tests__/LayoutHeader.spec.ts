import snapshowWrapper from "@/infra/__tests__/snapshotWrapper"
import LayoutHeaderVue from "../LayoutHeader.vue"
import router from "@/infra/router"

describe('infra/components/LayoutHeader', () => {
    it('should render', () => {
        expect(snapshowWrapper(LayoutHeaderVue, {
            plugins: [router]
        })).toMatchSnapshot()
    })
})