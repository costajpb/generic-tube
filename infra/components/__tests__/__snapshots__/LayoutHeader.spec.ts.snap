// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`infra/components/LayoutHeader > should render 1`] = `
<header
  class="_container_6cc6f3"
>
  <h1
    class="_title_6cc6f3"
  >
     Generic Tube 
  </h1>
  <div
    class="_extra_6cc6f3"
  >
    
    
  </div>
</header>
`;
